scripts/snyk/remove-samples-workspace.js

```diff
@@ -0,0 +1,14 @@
+const {resolve} = require('path');
+const {readFileSync, writeFileSync} = require('fs');
+const {cwd} = require('process');
+
+const packageJsonPath = resolve(cwd(), 'package.json');
+
+const packageJSON = JSON.parse(readFileSync(packageJsonPath));
+packageJSON.workspaces = packageJSON.workspaces.filter((wp) =>
+  ['packages/headless', 'packages/atomic'].includes(wp)
```

Comment:
I would probably extract this into a constant at the top of the file then. Maybe we could even move all "project name" constants to a common JS file in the `scripts` directory. It could contain something like:
```js
const packagesToUpdateNpmTag = [
  'atomic',
  'auth',
  'bueno',
  // ...
] as const;

const packagesToScanWithSnyk = [
  'headless',
  'atomic'
] as const;
```